mooniefill.lic

```diff
@@ -0,0 +1,86 @@
+custom_require.call(%w[common common-moonmage])
+
+class MoonieFill
+
+  def initialize
+    @magic = ['Dawgolesh', 'Durgaulda', 'Ismenia','Yavash', 'Wolf', 'Cobra', 'Ox', 'Donkey', 'Coyote', 'Dove', 'Phoenix', 'Cow', 'Shrike', 'Jackal', 'Amlothi', 'Wren', 'Nightingale', 'Owl', 'Adder', 'Shrew', 'Toad']
+    @lore = ['Cobra', 'Donkey', 'Wren', 'Ismenia', 'Xibar', 'Raven', 'Dolphin', 'Albatross', 'Heron', 'Welkin', 'Phoenix', 'Shardstar', 'Weasel', 'King Snake', 'Hare', 'Brigantine', 'Scales', 'Amlothi', 'Verena']
+    @survival = ['Sun', 'Dolphin', 'Brigantine', 'Ox', 'Heart', 'Lion', 'Unicorn', 'Boar', 'Panther', 'Wolverine', 'Shark', 'Goshawk', 'Raccoon', 'Triquetra', 'Szeldia', 'Heron', 'Cow', 'Shrike', 'Ram', 'Vulture', 'Archer']
+    @offense = ['Szeldia', 'Er\'qutra', 'Boar', 'Panther', 'Wolverine', 'Shark', 'Triquetra', 'Weasel', 'Scales', 'Coyote', 'Owl', 'Shrew', 'Cat', 'Mongoose', 'Merewalda', 'Archer', 'Adder', 'Scorpion', 'Centaur', 'Viper', 'Spider']
+    @defense = ['Merewalda', 'Dawgolesh', 'Mongoose', 'Scorpion', 'Centaur', 'Lion', 'Goshawk', 'Raccoon', 'Morleena', 'Albatross', 'Welkin', 'Shardstar', 'King Snake', 'Hare', 'Dove', 'Nightingale', 'Katamba', 'Magpie', 'Vulture', 'Jackal', 'Giant']
+    @pools = ['Survival', 'Magic', 'Lore', 'Defense', 'Offense']
+    do_the_things
+  end
+
+
+  def fill_pools
+    @pools.each do |pool|
+      case DRC.bput("predict state #{pool}", /no understanding/, /feeble understanding/, 
+                /weak understanding/, /fledgling understanding/, /modest understanding/,  
+                /decent understanding/, /significant understanding/, /potent understanding/, 
+                /insightful understanding/, /powerful understanding/, /complete understanding/
+      )
+      when /complete understanding/
+        DRC.message("#{pool} is full!")
+      else
+        observe(pool)
+      end
+    end
+  end
+
+  def observe(pool)
+    if pool == 'Survival'        
+      constellations = @survival
+    elsif pool == 'Magic'
+      constellations = @magic
+    elsif pool == 'Lore'
+      constellations = @lore
+    elsif pool == 'Offense'
+      constellations = @offense
+    elsif pool == 'Defense'
+      constellations = @defense
+    else
+      DRC.message("Wut?")
+    end
+
+    unless DRCMM.get_telescope?(settings.telescope_name, settings.telescope_storage)
```

Comment:
If we're using `settings.telescope_name` here, then I think we also need to use it on lines 51 and 52